src/angular-core/styles/typography/_scrollbar.scss

```
@@ -26,6 +26,16 @@ $trackColor: $sbbColorMilk;
       height: 0.5rem;
     }
 
+    @supports (-webkit-touch-callout: none) {
+      // Only visible on iOS devices.
```

This seems to be an odd phrasing for something hidden on iOS. See my suggestion or correct me, if I mistunderstood it.
```suggestion
      // Only applied on iOS devices.
```